docs/src/content/docs/guides/customization.mdx

```diff
@@ -0,0 +1,411 @@
+
+<Tabs>
+  <TabItem label="Frontmatter">
+
+```md
+---
+# src/content/docs/example.md
+title: Page with only H2s in the table of contents
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 2
+---
+```
+
+  </TabItem>
+  <TabItem label="Global config">
+
+```js
+// astro.config.mjs
+
+defineConfig({
+  integrations: [
+    starlight({
+      title: '',
+      tableOfContents: { minHeadingLevel: 2, maxHeadingLevel: 2 },
+    }),
+  ],
+});
+```
+
+  </TabItem>
+</Tabs>
+
+Disable the table of contents entirely by setting the `tableOfContents` option to `false`:
+
+<Tabs>
+  <TabItem label="Frontmatter">
+
+```md
+---
+# src/content/docs/example.md
+title: Page without a table of contents
+tableOfContents: false
+---
+```
+
+  </TabItem>
+  <TabItem label="Global config">
+
+```js
+// astro.config.mjs
+
+defineConfig({
+  integrations: [
+    starlight({
+      title: 'Docs with table of contents disabled globally',
+      tableOfContents: false,
+    }),
+  ],
+});
+```
+
+  </TabItem>
+</Tabs>
+
+## Social links
+
+Starlight has built-in support for adding links to your social media accounts to the site header via the [`social`](/reference/configuration/#social) option in the Starlight integration.
+
+Currently Discord, GitHub, Mastodon, and Twitter are supported.
+Let us know on GitHub or Discord if you need support for another service!
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+
+export default defineConfig({
+  integrations: [
+    starlight({
+      title: 'Docs With Social Links',
+      social: {
+        discord: 'https://astro.build/chat',
+        github: 'https://github.com/withastro/starlight',
+        mastodon: 'https://m.webtoo.ls/@astro',
+        twitter: 'https://twitter.com/astrodotbuild',
+      },
+    }),
+  ],
+});
+```
+
+## Edit links
+
+Starlight can display an “Edit page” link in each page’s footer.
+This makes it easy for a reader to find the file to edit to improve your docs.
+For open source projects in particular, this can help encourage contributions from your users.
+
+To enable edit links, set [`editLink`](/reference/configuration/#editlink) in the Starlight integration config.
+The value of `editLink.baseUrl` will be prepended to the path to the current page to form the full edit link.
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+
+export default defineConfig({
+  integrations: [
+    starlight({
+      title: 'Docs With Edit Links',
+      editLink: {
+        baseUrl: 'https://github.com/withastro/starlight/edit/main/',
+      },
+    }),
+  ],
+});
+```
+
+## Custom CSS styles
+
+Customize the styles applied to your Starlight site by providing additional CSS files to apply, modifying or extending Starlight’s default styles.
```

Comment:
I got a bit confused while reading this sentence, I believe that's because of the double "apply" in the same sentence and the drastic verb change from "apply" (Infinitive) to "modifying" (Present Continuous).

Maybe changing the wording along these lines could work?
```suggestion
Customize the styles applied to your Starlight site by providing additional CSS files to modify or extend Starlight’s default styles.
```